src/components/KSkeleton/KSkeletonBox.vue

```diff
@@ -39,8 +39,9 @@ defineProps({
     )
     repeat;
   background-size: 400% 100%;
-  border-radius: 3px;
+  border-radius: var(--kui-border-radius-10, $kui-border-radius-10);
   display: inline-flex;
+  margin-right: var(--kui-space-20, $kui-space-20) !important;
   transition: 1s;
```

Comment:
For review: decided to leave some of the values untouched here (animation timing, gradient) as they are very specific to loader component, rather than our UIs so I don't see value in tokenizing those. Open to feedback.